examples/placeholder/linux/include/static-supported-temperature-levels.h

```diff
@@ -0,0 +1,69 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/clusters/temperature-control-server/supported-temperature-levels-manager.h>
+#include <app/util/af.h>
+#include <app/util/config.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace TemperatureControl {
+
+/**
+ * This implementation statically defines the options.
+ */
+
+class StaticSupportedTemperatureLevels : public SupportedTemperatureLevelsIterator
+{
+    struct EndpointPair
+    {
+        EndpointId mEndpointId;
+        chip::CharSpan * mTemperatureLevels;
+        int mSize;
+
+        EndpointPair(EndpointId aEndpointId, chip::CharSpan * aTemperatureLevels, int aSize) :
+            mEndpointId(aEndpointId), mTemperatureLevels(new chip::CharSpan[aSize]), mSize(aSize)
```

Comment:
What I meant was: this is a copy/paste of the code from all-clusters-app.  All the problems that code had this code has too.  

Resolving comments with "I don't understand the comment" is a very strange thing to do.  If you don't understand, leave it unresolved so the conversation can happen, please.

Reply:
> Resolving comments with "I don't understand the comment" is a very strange thing to do. If you don't understand, leave it unresolved so the conversation can happen, please.

Oops!! I mistakenly resolved the thread

Reply:
Done